### PR TITLE
Description for Test Cases of Fb Audience Updated and Coverage Improved

### PR DESCRIPTION
**Pull request by anantjain45823**
Description of Test Cases updated and Coverage improved

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [x] Lint rules pass locally
- [x] The code changed/added as part of this pull request has been covered with tests
- [x] All tests related to the changed code pass in development

### Code review 

- [x]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [x] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: __tests__/data/fb_custom_audience.json [hunk 1]
```diff
@@ -1,6 +1,730 @@
 [
   {
-    "description": "Test Case 1",
+      "description": "No Message Event",
```
Comment (shrouti1507) [inline __tests__/data/fb_custom_audience.json:3]: No message `Type`  should be written ideally

File: __tests__/data/fb_custom_audience.json [hunk 2]
```diff
@@ -1,6 +1,730 @@
...
+          "error": "Message Type is not present. Aborting message."
+        }
+  },
+  {
+      "description": "Unsupported Event type ",
+      "input": {
+        "message":{
+          "userId": "user 1",
+          "anonymousId": "anon-id-new",
+          "type":"NAN",
```
Comment (shrouti1507) [inline __tests__/data/fb_custom_audience.json:94]: Make something like track or identify. Which is our valid event type

File: __tests__/data/fb_custom_audience.json [hunk 3]
```diff
@@ -189,8 +849,8 @@
       }
     ]
   },
-  {
-    "description": "Test Case 2",
+{
+    "description": "Type Unknown ",
```
Comment (shrouti1507) [inline __tests__/data/fb_custom_audience.json:853]: also mention that hashing is disabled as well

**Review by utsabc — APPROVED** [review]

File: __tests__/data/fb_custom_audience.json [hunk 4]
```diff
@@ -1,6 +1,730 @@
...
+          "Transformations": [],
+          "IsProcessorEnabled": true
+        },
+        "libraries": [],
+        "request": {
+          "query": {}
+        }
+      },
+      "output": {
+          "error": " NAN call is not supported "
```
Comment (shrouti1507) [inline __tests__/data/fb_custom_audience.json:172]: This test case is failing, Please change it to `Track call is not supported`

Comment (anantjain45823) [reply __tests__/data/fb_custom_audience.json]: Yeah. Updated

File: __tests__/data/fb_custom_audience.json (hunk 3 above)
Comment (shrouti1507) [reply __tests__/data/fb_custom_audience.json]: addressed hence closing

File: __tests__/data/fb_custom_audience.json (hunk 2 above)
Comment (shrouti1507) [reply __tests__/data/fb_custom_audience.json]: added , closing

**Review by shrouti1507 — APPROVED** [review]

**Review by utsabc — APPROVED** [review]

**Pull request merged**